core/src/test/java/com/google/googlejavaformat/java/DiagnosticTest.java

```diff
@@ -26,6 +26,9 @@
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.util.Locale;
+import org.junit.After;
+import org.junit.Before;
 import org.junit.Rule;
```

Comment:
I used google-java-format command line util on this file specifically against this file this time. It seems the import rules do allow wild card imports. I had to re-import every junit dep manually.

Reply:
> It seems the import rules do allow wild card imports.

Oh, really? It looks to me that point [3.3.1](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports) of the guide says that "**Wildcard imports**, static or otherwise, **are not used**."

Or have I misunderstood what you mean by "import rules"?

Reply:
Oh. I mean the rules coded in the source of google-java-format. I'm not familiar with the google java style guide. I am a fan of tools. That's why I came to google-java-format.

Reply:
Oh, how funny. I just tried it myself, and indeed google-java-format doesn't reformat wildcard imports as non-wildcard imports. I must've got confused with how google-java-format automatically removes unused imports. Apologies!

But regardless, thank you very much for removing the wildcards that were here previously. :)

Reply:
No problem :)